distributed/distributed.yaml

```
@@ -310,7 +310,9 @@ distributed:
       interval: 500ms
       disk: true  # Monitor host-wide disk I/O
       host-cpu: false  # Monitor host-wide CPU usage, with very granular breakdown
-      gil-contention: false  # Monitor GIL contention
+      gil-contention: 
+        enabled: false  # Monitor GIL contention
+        interval-microseconds: 1000  # Time in microseconds to poll GIL
```

Let's call this `interval`, instead of `interval-microseconds`, and accept string inputs like `"50ms"`, `"1 hour"`, etc. by using our `parse_timedelta` utility. This will be more consistent with how we handle other similar configurable time intervals.

```suggestion
        interval: "1ms"  # Time interval to poll the GIL
```

Here's one example of us using `parse_timedelta`

https://github.com/dask/distributed/blob/89d5ad4f462fe4cc50f975c2e10f6421fabd7c79/distributed/client.py#L936-L937

but grepping through the codebase, you'll find many other examples